cli/common/lua.go

```diff
@@ -66,20 +72,17 @@ func LuaGetRocksVersions(appDirPath string) (map[string]string, error) {
 			} else {
 				depInfoLTable.ForEach(func(depVersionL lua.LValue, _ lua.LValue) {
 					depVersion := depVersionL.String()
-					if _, found := rocksVersionsMap[depName]; found {
-						log.Warnf(
-							"Found multiple versions for %s dependency in rocks manifest",
-							depName,
-						)
+					if _, found := rocksInfo.Versions[depName]; found {
+						rocksInfo.Duplicates = append(rocksInfo.Duplicates, depName)
 					}
-					rocksVersionsMap[depName] = depVersion
+					rocksInfo.Versions[depName] = depVersion
```

Comment:
Are you guarantee that the newest version of the rock is shown? 

Should we show all present versions when showing `Duplicates`? I'm not sure but maybe it can be like this:

```
Rocks
 myapp scm-1
 errors 2.1.5-1
 frontend-core 7.5.0-1
 membership 2.3.2-1
 checks 3.1.0-1
 vshard 0.1.17-1
 metrics 0.9.0-1
 cartridge-cli-extensions 1.1.1-1
 graphql 0.1.1-1
 ddl 1.4.0-1
 http 1.1.0-1
 luagraphqlparser 0.1.0-1

   • Duplicate rocks found: cartridge (2.5.0, 2.6.0, 2.6.1), ddl (1.3.0, 1.3.1)

```